rclpy/rclpy/impl/rmw_implementation_tools.py

```diff
@@ -72,17 +75,48 @@ def import_rmw_implementation():
 
     If :py:func:`select_rmw_implementation` has not previously been called then
     a default implementation will be selected implicitly before loading.
+
+    :raises: :py:class:`NoImplementationAvailableException` if there are no
+        implementations available.
     """
     global __rmw_implementation_module
     if __rmw_implementation_module is not None:
         return __rmw_implementation_module
+    available_implementations = get_rmw_implementations()
     if __selected_rmw_implementation is None:
         logger = logging.getLogger('rclpy')
-        select_rmw_implementation(get_rmw_implementations()[0])  # select the first one
+        if available_implementations:
+            select_rmw_implementation(available_implementations[0])  # select the first one
+        else:
+            raise NoImplementationAvailableException()
```

Comment:
I don't think it makes a difference, but if you want to change it I'm fine with that.